stock_picking_invoicing_incoterm/models/stock.py

```diff
@@ -1,24 +1,12 @@
-# Copyright 2014-2015 Agile Business Group sagl
+# Copyright 2020 Agile Business Group sagl
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
-from openerp import api, fields, models
+from odoo import fields, models
 
 
 class StockPicking(models.Model):
     _inherit = "stock.picking"
 
-    incoterm = fields.Many2one(
-        comodel_name="stock.incoterms",
-        string="Incoterm",
-        help="International Commercial Terms are a series of predefined "
-        "commercial terms used in international transactions.",
+    incoterm_id = fields.Many2one(
```

Comment:
If you change the field's name, you should also add a migration script or the attribute `oldname` to the field's definition.

In order to avoid all of that, I recommend not to change the field's name.